### ISSUE TITLE
CSRF error

### ISSUE THREAD

**Issue by Traineratwot** [trypromptly/LLMStack#15]
I see CSRF error.

Where I can write CSRF_TRUSTED_ORIGINS ?

**Comment by ajhai** [comment]
@Traineratwot you can set `CSRF_TRUSTED_ORIGINS` as a comma separated list of strings in your `.env` file. So setting `CSRF_TRUSTED_ORIGINS=https://gpt.************.site` in your .env and restarting `./run-llmstack.sh` should get it work. Please let me know if this doesn't work.

**Comment by Traineratwot** [comment]
Yes, it helped, thank you. You should add this to the documentation

**Comment by ajhai** [comment]
Yeah just added it to the documentation. Thanks for opening the issue.
